pyiron_atomistics/atomistics/structure/periodic_table.py

```diff
@@ -214,15 +215,18 @@ def __init__(self, file_name=None):  # PSE_dat_file = None):
         self._parent_element = None
         self.el = None
 
-    def __getattr__(self, item):
-        return self[item]
-
     def __getitem__(self, item):
         if item in self.dataframe.columns.values:
             return self.dataframe[item]
         if item in self.dataframe.index.values:
             return self.dataframe.loc[item]
 
+    def get(self, item, default=None):
+        value = self[item]
```

Comment:
Same here, errors of `self[item]` should be cought if a default is provided.

Reply:
In both `ChemicalElement` and `PeriodicTable` `__getitem__` never raises an exception, but returns `None` on undefined keys.